libcxxabi/src/aix_state_tab_eh.inc

```diff
@@ -145,8 +143,10 @@ struct FSMEntry {
     intptr_t nextStatePtr;
   };
   union {
-    // Address of the destructor function.
-    void (*destructor)(void*, size_t);
+    // Address of the destructor function with 1 argument.
+    void (*destructor)(void*);
+    // Address of the destructor function with 2 arguments.
+    void (*destructor2)(void*, size_t);
```

Comment:
I would give this a more descriptive name:
```suggestion
    void (*xlC_destructor)(void*, size_t);
```
alternative name bikeshed:

- legacy_destructor
- xlcxxabi_destructor
- xl_destructor

etc.

Reply:
Thanks, renamed as `xlCDestructor`.